locales/en/population.json

```diff
@@ -1,4 +1,25 @@
 {
+  "daily_view": {
+    "missing_value": "-",
```

Comment:
Having a look at the screenshots I'm wondering if placing a `-` is the best way to communicate that we don't have any values for this item yet.

It might be better to replace the data layouts with a single message saying that there are no values for that location and to include a call to action (I'm assuming to the edit step?) to encourage that data to be provided?

Reply:
Agree - dealing with this with content is better. 
We say "No moves" on the dashboard when there are no moves booked for that day, and "No Health information warnings" when no info has been added to the PER - I think matching this would be best. "No numbers entered"? 

Alternatively we could use it to promote adding the number - ie. we say "Warnings will display once a Person Escort Record has been completed" on the record page... so could be "To be completed"?

Yeap, there's an edit state to be added.

Reply:
>We say "No moves" on the dashboard when there are no moves booked for that day, and "No Health information warnings" when no info has been added to the PER - I think matching this would be best. "No numbers entered"?

Yep, I think this would be a nice pattern to re-use. And within that banner style we can add a call to action to "add numbers"?

Reply:
Yeah, that's exactly what it is with the edit pattern - green button

Reply:
To confirm: "No numbers entered"

Reply:
The dashboard with no moves is showing "0 outgoing moves". 

![Screenshot from 2020-11-16 16-22-06](https://user-images.githubusercontent.com/196695/99279306-00abe400-2828-11eb-8220-7164310fa191.png)

Also, I'm not sure what this would look like in practice as there are multiple rows of data - so I've currently hidden the sections with no numbers. It looks like this:

![Screenshot from 2020-11-16 16-19-14](https://user-images.githubusercontent.com/196695/99279070-b7f42b00-2827-11eb-918a-75db93558373.png)

@Andy-Millar Can you make a quick mockup of what you'd like to have displayed, and attach it to the Jira story - then I'll make it match up accordingly.

Reply:
> The dashboard with no moves is showing "0 outgoing moves".
> 
> Screenshot from 2020-11-16 16-22-06

- I thought we agreed to just go straight to the overview without the blue blocks?

![IPTC-MVP-00-01 Copy](https://user-images.githubusercontent.com/22559322/99376530-f7208b80-28bc-11eb-8962-f1f9e53ba3a5.png)

Reply:
When there are no numbers I think we should do this:

<img width="1017" alt="Screenshot 2020-11-17 at 10 13 55" src="https://user-images.githubusercontent.com/22559322/99377111-baa15f80-28bd-11eb-9eea-ef67247ebb1d.png">

Alternatively could do this, but I prefer the above option:

<img width="1016" alt="Screenshot 2020-11-17 at 10 14 14" src="https://user-images.githubusercontent.com/22559322/99377145-c5f48b00-28bd-11eb-9d1c-c0fe81487100.png">

Reply:
Could also argue that once the numbers are added, the 'Update numbers' button becomes a link - 'Change numbers' - more in line with what we do elsewhere

Reply:
ie:

<img width="1018" alt="Screenshot 2020-11-17 at 10 19 27" src="https://user-images.githubusercontent.com/22559322/99377685-66e34600-28be-11eb-8432-ffa9beb104ec.png">

Reply:
Resolving this thread based upon separate conversations with @teneightfive  and @Andy-Millar